### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by theInternauts**

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -23,6 +23,18 @@
...
 
+get '/doodle/:doodle_key' do
+  @doodle = Doodle.find_by_doodle_key(params[:doodle_key])
+  erb :doodle_img_detail
+end
+
+
+post '/save' do
+  new_key = SecureRandom.urlsafe_base64(12)
+  @doodle = Doodle.create!({doodle_key: new_key, doodle_data: params[:doodle_data], user_id: 1})
```
Comment (zspencer) [inline app.rb:34]: Why are we always saving them as `user_1`?

File: public/js/app_raw.js [hunk 2]
```diff
@@ -4,8 +4,41 @@ function colorSelector(){
...
+
+  //turn image into URL for doodle_data
+  var dataURL = canvas.toDataURL();
+  var input = document.getElementsByName("doodle_data")[0];
+  input.value = dataURL;
+  console.log(input);
+
+  //run the submit that the we blocked initially
+  this.submit();
+};
```
Comment (zspencer) [inline public/js/app_raw.js:23]: Nice functions.

File: app.rb (hunk 1 above)
Comment (DiscreteObject) [inline app.rb:34]: Temporary hardcoding before we had user registration and login working.

File: public/js/app_raw.js (hunk 2 above)
Comment (salarkhan) [inline public/js/app_raw.js:23]: took forever to realize getElementsByName returns an array

**Pull request merged**